state_machine/drivers/emulation/lib/pycubed.py

```diff
@@ -43,26 +44,35 @@ def __init__(self):
         self.c_state_err = 0
         self.c_boot = None
 
+        self._accel = (1.0, 2.0, 3.0)
```

Comment:
What is the purpose of having non-zero values here?

Reply:
I just set them to non zero values to get slightly more interesting results out of the b-cross controller, so I set acceleration and magnetometer to nonzero and nonparallel values.

Reply:
Good idea